content/java/concepts/errors/terms/concurrentmodificationexception/concurrentmodificationexception.md

```diff
@@ -0,0 +1,51 @@
+---
+Title: 'ConcurrentModificationException'
+Description: 'Occurs when item Removed or Add from Iterable content during Iteration.'
+Subjects:
+  - 'Code Foundations'
+  - 'Computer Science
+Tags:
```

Comment:
Please only use tags from [this list](https://github.com/Codecademy/docs/blob/main/documentation/tags.md).

Reply:
Sure.!